index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="/styles.css" />
+  <title>My Portifolio</title>
+</head>
+
+<body>
+  <header>
+    <nav class="flexr container">
+      <a href="#" class="link">
+        <img src="https://arsenethierry-dev.vercel.app/images/logo-light.svg" width="90px" height="60px"
+          alt="github icon" />
+      </a>
+      <img src="/images/menu button.png" width="18px" height="12px" id="menu" alt="menu icon" />
+    </nav>
+  </header>
+
+  <section class="container flexc" id="header">
+    <div class="flexc mt">
+      <h1>I’m Arsene. 
+        <br />
+        Glad to see you!
+      </h1>
+      <p>
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact
+        me.
+      </p>
+    </div>
+    <section class="flexc section2">
+      <h3><a href="#" class="link">LET’S CONNECT</a></h3>
```

Comment:
- [OPTIONAL ] Your `header` is not look the same as the Figma design, we can see on the Figma design below  the word see you,.. is not on the same line instead is down, also your `p` tag is not using the same fonts as the Figma design, kindly fix this for your design to match the original one so that it will look attractive :wink: 
Your design 
![Screenshot from 2023-04-29 03-33-12](https://user-images.githubusercontent.com/88894525/235277506-5e72d343-ca47-493b-a2db-dd9ad52da53f.png)

Figma Design 
![Screenshot from 2023-04-29 03-37-51](https://user-images.githubusercontent.com/88894525/235277558-69548f0b-5c4f-42bc-a1ca-a7fb55fa927d.png)